### PR TITLE
[#1951] Add userfeed frontend component

### PR DESCRIPTION
**Pull request by jiromaykin**
issue: https://taiga.maykinmedia.nl/project/open-inwoner/task/1951

This 'placeholder' template, showing 2 cards, is not dynamic yet! Depends on [Taiga issue #1760](https://taiga.maykinmedia.nl/project/open-inwoner/task/1760).

Design: https://www.figma.com/file/iKGhWhstaLIlFSaND2q7cE/OIP---Designs-(new)?type=design&node-id=717%3A12249&mode=dev

The Danger/Warning color is default for Actions - when back-end is done, the 2 cards can be merged into 1 with dynamic CSS classes.

➕ adding new pattern for H2's that need to have an indicator dot.

### CODE REVIEW

File: src/open_inwoner/components/templates/components/UserFeed/UserFeed.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% load i18n icon_tags %}
+
+{# if userfeed data is true then show component #}
+<section class="userfeed">
+    <h2 class="h2 {# if userfeed true #}indicator{# endif #}">
+        {% trans "Aanvragen updates" %}
+    </h2>
+    <div class="userfeed__summery">
```
Comment (pi-sigma) [inline src/open_inwoner/components/templates/components/UserFeed/UserFeed.html:8]: summery > summary

File: src/open_inwoner/components/templates/components/UserFeed/UserFeed.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <ul class="userfeed__list">
+            <li class="userfeed__list-item">
+                <p class="userfeed__list-paragraph">
+                    Bij <span>X</span> aanvraag{# if statuses > 1 #}aanvragen{# endif #} is uw actie nodig
```
Comment (pi-sigma) [inline src/open_inwoner/components/templates/components/UserFeed/UserFeed.html:12]: Perhaps:
```
Bij <span>{# num statuses #}</span> aanvraag{# if statuses > 1 #}en{# endif #} is uw actie nodig
```

File: src/open_inwoner/components/templates/components/UserFeed/UserFeed.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                </p>
+            </li>
+            <li class="userfeed__list-item">
+                <p class="userfeed__list-paragraph">
+                    Bij <span>X</span> aanvraag{# if statuses > 1 #}aanvragen{# endif #} is de status gewijzigd
```
Comment (pi-sigma) [inline src/open_inwoner/components/templates/components/UserFeed/UserFeed.html:17]: See previous

File: src/open_inwoner/components/templates/components/UserFeed/UserFeed.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                </p>
+            </li>
+        </ul>
+    </div>
+    <div class="card-container card-container--columns-2">
+        <a href="{{ userfeedbackend }}" class="card {# if action-needed true #}card--status{# endif #}">
+            <div class="card__body card__body--tabled">
+                <p class="p tabled__value">Bijzondere bijstand aanvragen</p>
```
Comment (pi-sigma) [inline src/open_inwoner/components/templates/components/UserFeed/UserFeed.html:25]: Are we still translating?

File: src/open_inwoner/components/templates/components/UserFeed/UserFeed.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+                </h2>
+                <span class="button button--icon-before button--transparent button--secondary">
+                {% icon icon="east" icon_position="after" primary=True outlined=True %}
+                </span>
+            </div>
+        </a>
+        {# Example of card without action status #}
+        <a href="{{ userfeedbackend }}" class="card">
+            <div class="card__body card__body--tabled">
+                <p class="p tabled__value">Bijzondere bijstand aanvragen</p>
```
Comment (pi-sigma) [inline src/open_inwoner/components/templates/components/UserFeed/UserFeed.html:37]: See previous

File: src/open_inwoner/scss/components/UserFeed/UserFeed.scss [hunk 6]
```diff
@@ -0,0 +1,93 @@
+.userfeed {
+  position: relative;
+
+  &__summery {
```
Comment (pi-sigma) [inline src/open_inwoner/scss/components/UserFeed/UserFeed.scss:4]: summary

File: src/open_inwoner/scss/components/UserFeed/UserFeed.scss [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+    background-color: #fff;
```
Comment (pi-sigma) [inline src/open_inwoner/scss/components/UserFeed/UserFeed.scss:5]: Don't we have a variable for white?

File: src/open_inwoner/scss/components/UserFeed/UserFeed.scss [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+    color: var(--color-red-notification);
+    display: block;
+    line-height: var(--font-line-height-body);
+    position: relative;
+    overflow: hidden;
+    z-index: 1002;
```
Comment (pi-sigma) [inline src/open_inwoner/scss/components/UserFeed/UserFeed.scss:11]: That's very specific. What's the rationale for this value (no objection, just curious)?

**Review by pi-sigma — CHANGES_REQUESTED** [review]

File: src/open_inwoner/components/templates/components/UserFeed/UserFeed.html (hunk 2 above)
Comment (jiromaykin) [reply src/open_inwoner/components/templates/components/UserFeed/UserFeed.html:12]: :-) This whole line will probably be completely replaced later on by the back-end (where the plural/single situation might also be handled from there) so this is not definite. I could just remove this and make it completely static.

File: src/open_inwoner/components/templates/components/UserFeed/UserFeed.html (hunk 5 above)
Comment (jiromaykin) [reply src/open_inwoner/components/templates/components/UserFeed/UserFeed.html:37]: (this template is not used anywhere right now, so no need for translation here)

File: src/open_inwoner/components/templates/components/UserFeed/UserFeed.html (hunk 4 above)
Comment (jiromaykin) [reply src/open_inwoner/components/templates/components/UserFeed/UserFeed.html:25]: This text will come from the back-end later on and will be completely different, depending on a case with a certain action-text, so right now it is all still static (this template is not used anywhere now, so no need for translation right here).

File: src/open_inwoner/scss/components/UserFeed/UserFeed.scss (hunk 8 above)
Comment (jiromaykin) [reply src/open_inwoner/scss/components/UserFeed/UserFeed.scss:11]: You mean the Z-index? That's indeed my mistake 😞 - the other stuff is needed to prepare positioning for the pseudo-element (= the pipe symbol between list-items).

**Review by pi-sigma — APPROVED** [review]

**Pull request merged**